### PR TITLE
Client can get articles from api

### PR DESCRIPTION
**Pull request by holgertidemand**
#### PT Story: https://www.pivotaltracker.com/story/show/156273522
Changes proposed in this pull request:

* Create ability for clients to get articles from the api
#### What I have learned working on this feature:
* Learned to setup serializer
* Testing request spec with JSON api rspec
* That you need a Api main controller

### CODE REVIEW

File: app/serializers/article_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class ArticleSerializer < ActiveModel::Serializer
+  include Rails.application.routes.url_helpers
+  attributes :title, :link
+
+  def link
+    Rails.application.routes.url_helpers.api_v1_article_path(object)
```
Comment (tochman) [inline app/serializers/article_serializer.rb]: If you include `Rails.application.routes.url_helpers` on top, you don't need to use this again. Also, try `api_v1_article_url` instead.

**Review by tochman — CHANGES_REQUESTED** [review]
Almost done...

File: app/serializers/article_serializer.rb (hunk 1 above)
Comment (holgertidemand) [reply app/serializers/article_serializer.rb]: We tried using ``` api_v1_article_url``` but unsuccessfully 
``` 
Failure/Error: Rails.application.routes.url_helpers.api_v1_article_url(object)
     
     ArgumentError:
       Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true
``` 
Can't seams to figure out how to fix that error. So we just went with ```api_v1_path``` instead

File: app/serializers/article_serializer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class ArticleSerializer < ActiveModel::Serializer
+  include   Rails.application.routes.url_helpers
+  attributes :title, :link
+
+  def link
+  api_v1_article_path(object)
+  end
```
Comment (AKidd95) [inline app/serializers/article_serializer.rb]: little format issue here

**Review by tochman — APPROVED** [review]
Okay. Ready to go.

**Review by AKidd95 — CHANGES_REQUESTED** [review]
Looks great, still, hate to disapprove on such a small thing but it needs to be done. good job

**Review by AKidd95 — APPROVED** [review]

**Pull request merged**